command/command_test.go

```diff
@@ -1,49 +1,108 @@
-package command
+package command_test
```

Comment:
Is this a best practice to put tests to a different package than testee?

Reply:
this way you do blackbox testing and enforce testing the exported functions only